bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/TripDetail.java

```diff
@@ -75,4 +75,25 @@ public long getDurationInMinutes() {
         return Duration.between(startTime, endTime).toMinutes();
     }
 
+    public Optional<Integer> getFuelConsumption() {
+        if (fuelConsumption != null) {
+            return Optional.of(fuelConsumption);
+        }
+        return Optional.empty();
```

Comment:
Wouldn't this work?
```suggestion
        return Optional.ofNullable(fuelConsumption);
```
If not, then this would work for sure, though I prefer the first option is it works.
```suggestion
        Integer fuelConsumption = this.fuelConsumption
        if (fuelConsumption != null) {
            return Optional.of(fuelConsumption);
        }
        return Optional.empty();
```

Reply:
Tried your first approach without success. The second one seems good.